spec/controllers/services_controller_spec.rb

```diff
@@ -1,83 +1,94 @@
-# -*- encoding : utf-8 -*-
-require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+require 'spec_helper'
```

Comment:
1-line commit messages are rarely useful, unless perhaps the change is a _really_ simple change.

This should explain _what_ has been refactored and _why_.

I have no idea what I'm looking for when reviewing this 😞

Reply:
After figuring out what's changed, here's what I'd have written for this commit message:

```
* Simplify `require` of spec helper; can just reference the file by name as its already in the load path
* Make the top level `describe` block describe only the class we're testing
* Group specs under `describe '#method_name' do` block
* Remove line breaks from spec descriptions
* Reduce line lengths
```

Reply:
sorry 😞 - I'll belatedly squash that in as penance (also practise)